yt_dlp/extractor/mave.py

```diff
@@ -0,0 +1,84 @@
+import json
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    get_element_by_id,
+)
+
+
+class MaveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<channel_id>[a-z]+)\.mave\.digital/ep-(?P<short_id>[0-9]+)'
+    _TESTS = [
+        {
+            'url': 'https://ochenlichnoe.mave.digital/ep-25',
+            'md5': 'aa3e513ef588b4366df1520657cbc10c',
+            'info_dict': {
+                'id': 'ochenlichnoe-25',
+                'title': 'Между мной и миром: психология самооценки',
+                'description': 'md5:83183d7002dc32fbebc3ccecd4a1ac03',
+                'thumbnail': r're:https?://.*\.jpg$',
+                'ext': 'mp3',
+                'channel': 'Очень личное',
+                'channel_id': 'ochenlichnoe',
+                'channel_url': 'https://ochenlichnoe.mave.digital/',
+            },
+        },
+        {
+            'url': 'https://budem.mave.digital/ep-12',
+            'md5': 'e1ce2780fcdb6f17821aa3ca3e8c919f',
+            'info_dict': {
+                'id': 'budem-12',
+                'title': 'Екатерина Михайлова: "Горе от ума" не про женщин написана',
+                'description': 'md5:d9ce1fc1fb5fc7b7a4e7a0b84a7861c3',
+                'thumbnail': r're:https?://.*\.jpg$',
+                'ext': 'mp3',
+                'channel': 'Все там будем',
+                'channel_id': 'budem',
+                'channel_url': 'https://budem.mave.digital/',
```

Comment:
when running these tests to update the data, I would just remove everything except `id` and `ext`, and the test script will output the rest of the keys+values for you